### PR TITLE
Added Mahabub card

### PR DESCRIPTION
**Pull request by prio101**
Added the card for Mahabub Islam Prio.

### CODE REVIEW

**Comment by BaseMax** [comment]
Thank you and welcome.
Please remove HTML comments at the Markdown. e.g: `<!-- LEAVE EMPTY OR PUT LINK TO YOUR PROFILE -->`

File: _networks/mahabub.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+permalink: /network/prio101/ <!-- GITHUB-USERNAME -->
+name: Mahabub Islam
+github: prio101
+website: http://mahabub.space/ <!-- LINK TO YOUR WEBSITE -->
+email: miprio101@gmail.com
+instagram: <!-- LEAVE EMPTY OR PUT LINK TO YOUR PROFILE -->
+twitter: https://twitter.com/prio_mahabub
+linkedin: https://www.linkedin.com/in/miprio101/
+facebook: <!-- LEAVE EMPTY OR PUT LINK TO YOUR PROFILE -->
+country: Bangladesh
```
Comment (jbampton) [inline _networks/mahabub.md:12]:
```suggestion
layout: network
permalink: /network/prio101/
name: Mahabub Islam
github: prio101
website: http://mahabub.space/
email: miprio101@gmail.com
instagram:
twitter: https://twitter.com/prio_mahabub
linkedin: https://www.linkedin.com/in/miprio101/
facebook:
country: Bangladesh
```

File: _networks/mahabub.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+location: Sirajganj
+role: Web Developer
+languages: ruby elixir js python3 css3 html5 <!-- YOU SHOULDN'T USE SPACE IN THE NAME OF THE PROGRAMMING LANGUAGE -->
+---
+
```
Comment (jbampton) [inline _networks/mahabub.md:17]:
```suggestion
```

File: _networks/mahabub.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+website: http://mahabub.space/ <!-- LINK TO YOUR WEBSITE -->
+email: miprio101@gmail.com
+instagram: <!-- LEAVE EMPTY OR PUT LINK TO YOUR PROFILE -->
+twitter: https://twitter.com/prio_mahabub
+linkedin: https://www.linkedin.com/in/miprio101/
+facebook: <!-- LEAVE EMPTY OR PUT LINK TO YOUR PROFILE -->
+country: Bangladesh
+location: Sirajganj
+role: Web Developer
+languages: ruby elixir js python3 css3 html5 <!-- YOU SHOULDN'T USE SPACE IN THE NAME OF THE PROGRAMMING LANGUAGE -->
```
Comment (jbampton) [inline _networks/mahabub.md:15]:
```suggestion
languages: ruby elixir js python3 css3 html5
```

File: _networks/mahabub.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+3. Code Review .
+4. UI+UX lover.
+5. Latest tools usage to make web app/mobile app optimization more comfortable .
+6. Linux Lover .
+7. Git
+8. Ruby, Ruby On Rails
+9. Elixir Lang, Phoenix
+10. React Js, Javascript
+11. Flutter / Dart
+I am moreover into latest technology and good parts of anything out there always learning . I make open source contributions frequently on GITHUB.
```
Comment (jbampton) [inline _networks/mahabub.md:32]:
```suggestion
1. Test Driven Development
2. Through On Documentation
3. Code Review
4. UI+UX lover
5. Latest tools usage to make web app/mobile app optimization more comfortable
6. Linux Lover
7. Git
8. Ruby, Ruby On Rails
9. Elixir Lang, Phoenix
10. React JS, JavaScript
11. Flutter / Dart
I am moreover into latest technology and good parts of anything out there always learning. I make open source contributions frequently on GitHub.
```

File: _networks/mahabub.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+twitter: https://twitter.com/prio_mahabub
+linkedin: https://www.linkedin.com/in/miprio101/
+facebook:
+country: Bangladesh
+location: Sirajganj
+role: Web Developer
+languages: ruby elixir js python3 css3 html5
+---
+
+Professional Web Developer and Web technology enthusiast . Working with web technology with passion for around 5 years. Here is some key skills I can point out about me,
```
Comment (jbampton) [inline _networks/mahabub.md:18]:
```suggestion
Professional Web Developer and Web technology enthusiast. Working with web technology with passion for around 5 years. Here is some key skills I can point out about me,
```

File: _networks/mahabub.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+3. Code Review .
+4. UI+UX lover.
+5. Latest tools usage to make web app/mobile app optimization more comfortable .
+6. Linux Lover .
+7. Git
+8. Ruby, Ruby On Rails
+9. Elixir Lang, Phoenix
+10. React Js, Javascript
+11. Flutter / Dart
+I am moreover into latest technology and good parts of anything out there always learning . I make open source contributions frequently on GITHUB.
```
Comment (jbampton) [inline _networks/mahabub.md:31]:
```suggestion
1. Test Driven Development
2. Through On Documentation
3. Code Review
4. UI+UX lover
5. Latest tools usage to make web app/mobile app optimization more comfortable
6. Linux Lover
7. Git
8. Ruby, Ruby On Rails
9. Elixir Lang, Phoenix
10. React JS, JavaScript
11. Flutter / Dart
I am moreover into latest technology and good parts of anything out there always learning. I make open source contributions frequently on GitHub.
```

**Review by jbampton — APPROVED** [review]

**Pull request merged**